### ISSUE TITLE
Add a `conda` install option for `deduplipy`

### ISSUE THREAD

**Issue by sugatoray** [fritshermans/deduplipy#15]
It will be nice to have a conda install option. I have started the work on adding this library to conda-forge channel already (PR: https://github.com/conda-forge/staged-recipes/pull/17495). Once the PR is merged, you will be able to install `deduplipy` with:

```sh
conda install -c conda-forge deduplipy
```

**Comment by sugatoray** [comment]
cc: @fritshermans 

Btw, I spotted `deduplipy` in this presentation of yours: https://youtu.be/SB1piCt6Vs0 :rocket: :star:

**Comment by fritshermans** [comment]
thanks for your work, I'll have a look at it in the coming days and get back to you

**Comment by fritshermans** [comment]
i see your PR is already merged but I'm not able to install DedupliPy with the command above. Does it work on your side?

**Comment by fritshermans** [comment]
any update on this @sugatoray ?

**Comment by sugatoray** [comment]
> i see your PR is already merged but I'm not able to install DedupliPy with the command above. Does it work on your side?

Thank you for checking it out. The reason this happened is because `deduplipy` had some other dependecies (as shown below) that were also not available on conda-forge and I had to create their recipes in the same PR. The dependecies got added to conda-forge (except one: `fancyimpute`, which had one of it's own dependency, `knnimpute` also added to conda-forge in the same PR). But `deduplipy` needs another manual push before it is available on conda-forge (this is only necessary once, if the dependencies are made in the same PR, while adding to conda-forge).

```sh
deduplipy
|--- pyminhash
|--- thefuzz
|--- fancyimpute
		|--- knnimpute
```

So, here's what I am going to do:

Initiate manual push in the following order: 

> 1. `fancyimpute`
> 2. `deduplipy`

Please try them tomorrow, they should be available.

**Comment by sugatoray** [comment]
Just pushed again. Should be available in 2-3 hours for downloading.

You can check if it is available with:

```sh
conda search -c conda-forge deduplipy
```

![image](https://user-images.githubusercontent.com/10201242/150595170-24e357f8-2f32-4cb5-857b-1b61a955b486.png)

**Comment by sugatoray** [comment]
As a next step to manage (update) the package on conda-forge, alongside your PyPI releases, I would suggest you to become a co-maintainer of the conda-forge feedstock: https://github.com/conda-forge/deduplipy-feedstock/. 

Currently, I am the sole maintainer. Typically, right after each of your PyPI releases, you could update a few fields in a recipe file (path: `recipe/meta.yaml`) in the feedstock, and wait to see if all the CI's completed the runs sucessfully. If they did, just merge the PR.

Updating the recipe file essentially entails:

fork the feedstock >> create new branch, such as `update_v0.7.5`; make changes and submit a PR.

**Comment by sugatoray** [comment]
So, if it's okay with you, I am going to add you as a co-maintainer there in the feedstock. When the next release happens on PyPI, please let me know. I will submit a PR to update the recipe there. And you could see how simple that typically is. Then during the next release, you could try updating.

Please let me know.
